test/e2e/BrsComponents.test.js

```diff
@@ -188,6 +188,35 @@ describe("end to end brightscript functions", () => {
         ]);
     });
 
+    test.only("components/roXMLElement.brs", () => {
```

Comment:
Including `test.only` will disable other tests in this file.  Please remove these once you're done debugging.  I'll fix this one before I merge.

```suggestion
    test("components/roXMLElement.brs", () => {
```